app/src/components/ChoiceBoard/useChoiceBoardModel.tsx

```diff
@@ -0,0 +1,31 @@
+import {useState} from 'react';
+
+export default (originalOptions) => {
+    // Selected Items
+    const [selectedItems, setSelectedItems] = useState(Array());
+
+    // All choices
+    const [allOptions, setAllOptions] = useState(originalOptions);
```

Comment:
Maybe, instead of receiving the `originalOptions` as parameter, we could add `useEffect` here that would fetch those `originalOptions` from the backend. 
This backend URL could be a environment param. If you are not used to environment params in ReactJS, let me know because I can help you with that.

What do you think?

Reply:
Tiago, 

Yes I think that is a fantastic idea. I have never looked at environment params before, but it will be another great learning opportunity! If I have any questions though I will be sure to let you know. Since this is my first pull request and I am unfamiliar ground, do I close this pull request, make my changes, then submit another pull request? Or do I somehow make the changes on this same pull request?

p.s. Thank you so much for being so positive and creating such an inclusive environment for first time contributors. 

Thanks,

Reply:
Hi there Reed! 👋 

About the PR, you don't need to close this and open another one. When you open a PR it doesn't take a snapshot of the branch/code in the moment of the PR and uses that "snapshot" to merge things; instead, it just creates a reference to the branch from which you are willing to pull code from. In this case here, as you (correctly) created a branch on your fork named "ChoiceBoard", and all the code you push to it before this PR is merged will be available to this PR. That's why it is a good practice to create different branches for issues, beacuse, this way, you could have multiple PR pointing to different "issue-branches" and you can work on each of them separately not messing up the specific codes of each work and having different development pace in each one.

About the environment var, take your time to read some references but to show you how we will use it in our project, I just pushed some code to your branch with this implementation (ChoiceBoard). My intention is to show you 3 things:
1) You can push code to the branch and it will be available in this PR ✅ 
2) I can commit directly to your branch as part of my reviewing (it's kinda pair programming thing) ✅ 
3) Simple example on how to use environment var ✅ 

Some tips:
1) In your commit messages, put a reference to the issue it is addressing so this trace will appear in the issue like this:

<img width="970" alt="Captura de Tela 2020-07-21 às 07 53 04" src="https://user-images.githubusercontent.com/3986989/88047152-8c865b80-cb27-11ea-921f-b5fea0cbdd0d.png">

2) Name your branch with the issue number prefix, like `issue-34-ChoiceBoard` or just `issue-34`


PS - You are welcome and I thank you so much for being such an enthusiastic member that is bring real value to this product.! 🖖

Reply:
Tiago, 

After looking into environment variables I believe that I see where you implemented them in the package.json file:

"dev": "REACT_APP_BACKEND_URL=http://localhost:8008 yarn start",
"build": "REACT_APP_BACKEND_URL=BUILD_APP_BACKEND_URL react-scripts build",

Therefore, what would be the exact syntax to access the REACT_APP_BACKEND_URL? Would it be process.REACT_APP_BACKEND_URL or process.env.REACT_APP_BACKEND_URL? 

Thank you for your help,

Reply:
It will be `process.env.REACT_APP_BACKEND_URL`

Reply:
Perfect, thank you so much for your help. I have pushed up the code that contains these new changes, please have a look.

Thank you, 

Reed Hopkins